patches/@deadcanaries+granax+3.2.5.patch

```diff
@@ -67,7 +48,72 @@ index edd3ca0..e606b93 100644
        default:
          throw new Error(`Unsupported platform "${platform}"`);
      }
-@@ -261,7 +262,7 @@ exports.install = function(callback) {
+@@ -136,9 +137,28 @@ exports._unpackMacintosh = function(bundle, callback) {
+     path.join(BIN_DIR, '.tbb.dmg')
+   ], { cwd: BIN_DIR });
+ 
++  const finalback = (err, value) => {
++    const extract = childProcess.spawn('hdiutil', [
++      'detach',
++      path.join(BIN_DIR, '.tbb')
++    ], { cwd: BIN_DIR });
++
++    extract.on('close', (code) => {
++      if (code < 0) {
++        const unpackErr = new Error('Failed to unpack bundle');
++        if (err) {
++          unpackErr = new AggregateError([unpackErr, err], 'Multiple errors attempting to unpack Tor');
++        }
++        return callback(unpackErr);
++      }
++
++      callback(null, value);
++    });
++  };
++
+   mounter.on('close', (code) => {
+     if (code < 0) {
+-      return callback(new Error('Failed to unpack bundle'));
++      return finalback(new Error('Failed to unpack bundle'));
+     }
+ 
+     ncp.ncp(
+@@ -146,21 +166,10 @@ exports._unpackMacintosh = function(bundle, callback) {
+       path.join(BIN_DIR, '.tbb.app'),
+       (err) => {
+         if (err) {
+-          return callback(new Error('Failed to unpack bundle'));
++          return finalback(new Error('Failed to unpack bundle'));
+         }
+ 
+-        const extract = childProcess.spawn('hdiutil', [
+-          'detach',
+-          path.join(BIN_DIR, '.tbb')
+-        ], { cwd: BIN_DIR });
+-
+-        extract.on('close', (code) => {
+-          if (code < 0) {
+-            callback(new Error('Failed to unpack bundle'));
+-          }
+-
+-          callback(null, getTorPath('darwin'));
+-        });
++        finalback(null, getTorPath('darwin'));
+       }
+     );
+   });
+@@ -242,7 +251,9 @@ exports.install = function(callback) {
+           const source = path.dirname(granax.tor(os.platform()));
+           const dest = path.join(BIN_DIR, 'Tor');
+ 
+-          console.log(`Moving tor binary and libs to ${dest}...`);
++          console.log(`Moving tor binary and libs from ${JSON.stringify(source)} to ${JSON.stringify(dest)}...`);
++          // Just in case a prior exists:
++          rimraf.sync(dest);
```

Comment:
Noticing that this PR deleting the Linux `Tor` binaries from the patch, this line is not really necessary.